src/components/BoxColor.jsx

```diff
@@ -0,0 +1,27 @@
+function BoxColor({ r, g, b }) {
+  const bgColor = {
+    backgroundColor: `rgba(${r},${g},${b})`,
+  };
+
+  const rgbToHex = (r, g, b) =>
+    '#' +
+    [r, g, b]
+      .map((x) => {
+        const hex = x.toString(16);
+        return hex.length === 1 ? '0' + hex : hex;
+      })
+      .join('');
+
+  return (
+    <div>
+      <div
+        className="card fs-4 w-25 border border-3 border-dark m-3 p-3 text-center"
```

Comment:
Aqui te faltaria añadir una cosa muy tonta, si te fijas el color de la letra en el rojo es blanca y en el del verde es negro, si pones lo siguiente lo tendrias :D, valiendonos del bootstrap :D:D
```suggestion
        className=`card fs-4 w-25 border border-3 border-dark m-3 p-3 text-center ${g>0 ? "test-black" : "text-white"}`
```